ioc.types.ts

```diff
@@ -0,0 +1,24 @@
+const TYPES = {
+    AuthManager: Symbol('AuthService'),
+    Bucket: Symbol('Bucket'),
+    Container: Symbol('Container'),
+    DataReader: Symbol('IngestionReader'),
+    DataWriter: Symbol('IngestionWriter'),
+    DummyHandler: Symbol('DummyHandler'),
+    DummyMessage: Symbol('DummyMessage'),
+    INGEST: Symbol('Ingest'),
+    Logger: Symbol('Logger'),
+    MessageFactory: Symbol('MessageFactory'),
+    MessageHandlerFactory: Symbol('MessageHandlerFactory'),
+    PREVIEW: Symbol('Preview'),
+    PreviewHandler: Symbol('PreviewHandler'),
+    PreviewMessage: Symbol('PreviewMessage'),
+    QueueUrl: Symbol('QueueUrl'),
+    S3Config: Symbol('S3Config'),
+    SQSConfig: Symbol('SQSConfig'),
```

Comment:
Is it expected to see both QueueUrl and SQSConfig here? Don't they express two levels of the same configuration?